docs/openapi/resources/presentation-submissions.yml

```
@@ -4,7 +4,6 @@ post:
   description: End a presentation exchange flow
   tags:
     - Presentations
```

```suggestion
    - Presentations
        security:
            - OAuth2: [submit:presentations]
```

might be more inline with the spec?

---

@mprorock, `/presentations/available` and `/presentations/submission` MUST NOT require OAuth2, see #242 and #243.

---

+1 switching to approved - my bad